editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -1,394 +1,374 @@
-/*******************************************************************************
- * Copyright (c) 2016 Red Hat, Inc.
- * Distributed under license by Red Hat, Inc. All rights reserved.
- * This program is made available under the terms of the
- * Eclipse Public License v1.0 which accompanies this distribution,
- * and is available at http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- * Red Hat, Inc. - initial API and implementation
- ******************************************************************************/
-package org.fusesource.ide.projecttemplates.wizards;
-

```

Comment:
same issue than for Brian.
I previously created those files with Windows line ending unfortunately, we configure to use Linux line-ending now.